Packages/Status/Sources/Status/Row/StatusRowView.swift

```diff
@@ -453,43 +453,88 @@ public struct StatusRowView: View {
   }
 
   @ViewBuilder
-  private var trailinSwipeActions: some View {
-    Button {
-      Task {
-        HapticManager.shared.fireHaptic(of: .notification(.success))
+  private var trailingSwipeActions: some View {
+    if preferences.swipeActionsStatusRight1 != StatusAction.none {
+      makeSwipeButton(action: preferences.swipeActionsStatusRight1)
+        .tint(theme.tintColor)
+    }
+    if preferences.swipeActionsStatusRight2 != StatusAction.none {
+      makeSwipeButton(action: preferences.swipeActionsStatusRight2)
+        .tint(.gray)
+    }
+  }
+
+  @ViewBuilder
+  private var leadingSwipeActions: some View {
+    if preferences.swipeActionsStatusLeft1 != StatusAction.none {
+      makeSwipeButton(action: preferences.swipeActionsStatusLeft1)
+        .tint(theme.tintColor)
+    }
+    if preferences.swipeActionsStatusLeft2 != StatusAction.none {
+      makeSwipeButton(action: preferences.swipeActionsStatusLeft2)
+        .tint(.gray)
+    }
+  }
+  
+  @ViewBuilder
```

Comment:
Since we only have a single view being returned here, we can safely drop `@ViewBuilder`.